### DISCUSSION TITLE
Unblocked everything (if you dont have a allow list)

### DISCUSSION THREAD

**Discussion by phillipbruh** [3kh0/ext-remover#561] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
First goto chrome://settings/signOut
(with a capital O)
and click signout

then goto tinyurl.com/addsession
add your personal account

then you can goto the chrome webstore and install somthing like leaf browser or canvas helper

**Comment by Aka-but-nice** [comment]
I'm deleting this, this is Killcurly, and this is stated on the home page.

**Reply by S-PScripts** [reply]
you locked it , not deleted it...
